rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton-v3.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <linux/types.h>
+#include <sys/stat.h>
+#include "liberpal-skeleton.h"
+#include "sgx_call.h"
+#include "defines.h"
+
+int pal_get_version(void)
+{
+	return 3;
+}
+
+int pal_init(pal_attr_t *attr)
+{
+	return __pal_init(attr);
+}
+
+int pal_create_process(pal_create_process_args *args)
+{
+	if (args == NULL || args->path == NULL || args->argv == NULL || args->pid == NULL || args->stdio == NULL) {
+		errno = EINVAL;
+		return -1;
+	}
+
+	if (!is_oot_driver) {
+		return __pal_create_process(args);
+	}
+
+	return 0;
+}
+
+int pal_exec(pal_exec_args *attr)
+{
+	if (attr == NULL || attr->exit_value == NULL) {
+		errno = EINVAL;
+		return -1;
+	}
+
+	if (!is_oot_driver) {
+		__pal_exec_v2(attr);
+	}
+
+	/* the `rune attest` and `rune exec` commands both need skeleton bundles/containers keeping running state.
+	* Therefore, we use a main loop to keep skeleton bundles/containers running.
+	*/
+	while(1) {
+		printf("Hello Skeleton!\n");
+		printf("    - Powered by ACK-TEE and runE\n");
+		fflush(stdout);
+		sleep(3);
+	}
```

Comment:
I don't like this way for liveness of skeleton enclave runtime. You always have enclave_runtime.args to do this.